README.md

```diff
@@ -63,7 +63,7 @@ To run nginx proxy as a separate container you'll need:
 curl https://raw.githubusercontent.com/jwilder/nginx-proxy/master/nginx.tmpl > /path/to/nginx.tmpl
 ```
 
-2) Set the `NGINX_DOCKER_GEN_CONTAINER` environment variable to the name or id of the docker-gen container.
+2) Use the `com.github.jrcs.letsencrypt_nginx_proxy_companion.docker_gen=true` label on the docker-gen container, or explicitly set the `NGINX_DOCKER_GEN_CONTAINER` environment variable to the name or id of that container.
```

Comment:
Actually there is, because it defaults to an empty string and I kept the logic from #181 where it checks for the value "true": `jq -r '.[] | select( .Labels["'$1'"] == "true")|.Id'`

We would have to change this query, I'm not opposed to it.